csrc/device_lower/lower2device.cpp

```diff
@@ -391,7 +391,7 @@ void GpuLower::analysis(Fusion* fusion) {
   // functionality should be affected. New IterDomains may be created,
   // so it is expected that generated code may use diffrent variable
   // names
-  if (isOptionEnabled(EnableOption::IdModel)) {
```

Comment:
@naoyam Is this change intended? #2253 is now showing up in thunder's benchmark CI.

cc'ing @IvanYashchuk

Reply:
Yep, my investigation also led to this PR. I haven't taken a look at our nightly reports and we need to check whether this problem was caught there.